explore-cardano/understanding-the-delegation-mechanism.md

```diff
@@ -1,5 +1,34 @@
+
+Stake delegation gives rise to “stake pools” that act in the similar way to mining pools in the Bitcoin protocol. Stake pool operators must be online in order to generate blocks when they are selected as slot leaders.
+
+## Stake delegation requirements
+
+Delegating stake requires posting two certificates to the chain: a staking key registration, and a delegation certificate. Posting certificates requires funds, so a user setting up their first wallet will need a bootstrapping mechanism. This mechanism relies on the possibility of base addresses using a staking key before posting the registration certificate for that key.
+Delegation scheme
+
+With the concept of delegation, any stakeholder can allow a stake pool to generate blocks for the Cardano network and then the rewards will be paid by the protocol for all participants including the fees for the stake pool operators. A stakeholder can transfer the right to generate blocks by creating a proxy signing key that allows the delegate to sign messages.
+
+In order to limit the delegate’s block generation power to a certain range of epochs and slots, the stakeholder can limit the proxy signing key’s valid message space to strings ending with a slot number in a specific range of values. This simple scheme is secure due to the verifiability and prevention of misuse properties of proxy signature schemes. This ensures that any stakeholder can verify that a proxy signing key was actually issued by a specific stakeholder to a specific delegate and that the delegate can only use these keys to sign messages inside the key’s valid message space, respectively.
+
+A stakeholder signs a certificate specifying the delegate’s identity (e.g., their public key) and the valid message space. Later on, the delegate can sign messages within the valid message space by providing signatures for these messages under its own public key along with the signed certificate.
+
+## Stake delegation scenario
+
+Imagine a user who is about to receive their first ada,for example through redemption, from a trade on an exchange, or some other source. They will set up a new wallet, and create an address to receive those funds. This address will be a base address, using a staking key that is generated by the wallet, but not yet registered on the chain.
+
+After receiving the initial funds, the user can then participate in staking, by posting a staking key registration certificate, as well as a delegation certificate for their staking key. Once the key is registered, newly created addresses can be pointer addresses to the staking key registration certificate.
+
+## Eligibility of stake delegation
+
+Delegation as described above can ameliorate fragmentation that may occur in the stake distribution. Nevertheless, this does not prevent a malicious stakeholder from dividing its stake to multiple accounts and, by refraining from delegation, induce a very large committee size.
```

Comment:
I do not see anything malicious with and individual splitting their stake up into multiple stake credentials. By "refraining from delegation" are you referring to the individual instead becoming an owner of a pool? A large committee size is perfectly fine (if you meant the number of owners).